src/components/mass_properties.rs

```diff
@@ -193,6 +193,11 @@ pub struct CenterOfMass(pub Vector);
 impl CenterOfMass {
     /// A center of mass set at the local origin.
     pub const ZERO: Self = Self(Vector::ZERO);
+
+		/// Get the inner [`Vector`] in local space of the [`CenterOfMass`]
+		pub fn inner(&self) -> Vector {
+			self.0
+		}
```

Comment:
This feels very unnecessary, you can just do `center_of_mass.0` like with all other components

Reply:
Yes I agree it is unnecessary
I do like the ability to put documentation on it though, but will remove at your request